sdk/translation/azure-ai-translation-document/tests/asynctestcase.py

```diff
@@ -3,6 +3,7 @@
 # Licensed under the MIT License.
 # ------------------------------------
 
+from azure.ai.translation.document.models._models import BatchRequest, SourceInput, StartTranslationDetails
```

Comment:
We should be able to just import from `azure.ai.translation.document.models`

Reply:
Updated